src/shapes/group.class.js

```diff
@@ -383,8 +383,7 @@
      * Execute the drawing operation for an object on a specified context
      * @param {CanvasRenderingContext2D} ctx Context to render on
      */
-    drawObject: function (ctx) {
-      this._renderBackground(ctx);
```

Comment:
why don't you want this?
or you want it in the next PR?
it's instead of fill property.

Reply:
group does not have fill today.
I'm trying to separate refactor from new features where possible.
I'm trying to have a series of (not so) small PRs where in case of errors, we can trace back where the behavour changed and why.

If this PR can have group with the new code with the listener objects, and all the other stuff working as. it was working before, we made a huge step forward

Reply:
Alright